vllm/model_executor/models/minicpmv.py

```diff
@@ -966,8 +908,7 @@ def init_llm(
         vllm_config: VllmConfig,
         prefix: str = "",
     ) -> nn.Module:
-        return LLMWrapper(Qwen2Model(vllm_config=vllm_config, prefix=prefix),
-                          name="model")
+        return Qwen2ForCausalLM(vllm_config=vllm_config, prefix=prefix)
```

Comment:
I remember that  I introduced LLMWrapper specifically to ensure proper loading of LoRA weights. Have you considered this?

Reply:
Yes. Since `Qwen2Model` is in `Qwen2ForCausalLM.model`, the weight loading should still work.

Reply:
Make sense